xwiki-rendering-transformations/xwiki-rendering-transformation-macro/src/main/java/org/xwiki/rendering/internal/macro/DefaultMacroManager.java

```diff
@@ -99,8 +99,8 @@
             } catch (ParseException e) {
                 // One of the macros is registered against the component manager with an invalid macro id, ignore it
                 // but log a warning.
-                this.logger.warn("Invalid Macro descriptor format for hint [" + entry.getKey()
-                    + "]. The hint should contain either the macro name only or the macro name followed by "
+                this.logger.warn("Invalid Macro descriptor format for hint [ {}" + entry.getKey()
```

Comment:
Actually a better fix would be to keep the `{}` (which i the slf4j syntax for a place holder) and move `entry.getKey()` as a second parameter.

Reply:
Done.